doc/build_apps/demo.rst

```diff
@@ -31,7 +31,7 @@ The following command will run a simple one node test network on a single machin
     [16:14:05.294] Starting 1 CCF node...
     [16:14:05.295] Virtual mode enabled
     [16:14:10.010] Started CCF network with the following nodes:
-    [16:14:10.011]   Node [0] = https://127.0.0.1:8000
+    [16:14:10.011]   Node [50211327a77fc16dd2fba8fae5fffac3df909fceeb307cf804a4125ae2679007] = https://127.0.0.1:8000
```

Comment:
Should this also be a limited prefix rather than the full digest? The interesting information on this line is the URL, and the prefix should be sufficient to go find this node's real ID if needed.

Reply:
Good point. I wrote this as the short hash at first but I believe that the short hash is for internal use only (e.g. it is the full hash that has to be included in a proposal to retire a node). But then the sandbox isn't used for node operation I don't think so maybe that's OK? Alternatively, I'd be happy to leave this as the `node.local_node_id` (i.e. `0`) too to avoid confusion. What do you think?

Reply:
As discussed, let's go for the local Python infra monotonic node ID as it is easier to find out in which directory the node is started.